### PR TITLE
Fix: samples DSN and missing inAppIncludes

### PR DESCRIPTION
**Pull request by marandaneto**
## :scroll: Description
Fix: samples DSN and missing inAppIncludes
Found 2 bugs #1293 and #1291

## :bulb: Motivation and Context
#1281

## :pencil: Checklist

- [X] I reviewed the submitted code
- [ ] I added tests to verify the changes
- [ ] I updated the docs if needed
- [X] No breaking changes

### CODE REVIEW

File: sentry-samples/sentry-samples-jul/src/main/resources/sentry.properties [hunk 1]
```diff
@@ -2,3 +2,5 @@
 dsn=https://502f25099c204a2fbf4cb16edc5975d1@o447951.ingest.sentry.io/5428563
 debug=true
 environment=staging
+# not tested yet, not reporting events at all
+in-app-includes=io.sentry.samples
```
Comment (marandaneto) [inline sentry-samples/sentry-samples-jul/src/main/resources/sentry.properties:6]: @maciejwalkowiak

File: sentry-samples/sentry-samples-log4j2/src/main/resources/sentry.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# not tested yet, not reporting events at all
+in-app-includes="io.sentry.samples"
```
Comment (marandaneto) [inline sentry-samples/sentry-samples-log4j2/src/main/resources/sentry.properties:2]: @maciejwalkowiak I'd guess solving https://github.com/getsentry/sentry-java/pull/1282/files#r580246878 would solve this one too

File: sentry-samples/sentry-samples-spring/src/main/resources/sentry.properties [hunk 3]
```diff
@@ -1 +1,3 @@
 debug=true
+# it didn't work, looks like every frame was inApp=false
+in-app-includes="io.sentry.samples"
```
Comment (marandaneto) [inline sentry-samples/sentry-samples-spring/src/main/resources/sentry.properties:3]: @maciejwalkowiak does spring (non boot) reads from external resources?

**Comment by marandaneto** [comment]
@maciejwalkowiak sentry-samples-servlet is not tested either, as there's no simple run and forget command like the others, is there a way to do like the other samples, so we can run thru Gradle instead of deploying the war file?

File: sentry-samples/sentry-samples-jul/src/main/java/io/sentry/samples/jul/Main.java [hunk 4]
```diff
@@ -23,5 +24,7 @@ public static void main(String[] args) {
     } catch (Exception e) {
       LOGGER.log(Level.SEVERE, "Something went wrong", e);
     }
+    // it didn't help either, looks like SDK is not even initializing.
```
Comment (maciejwalkowiak) [inline sentry-samples/sentry-samples-jul/src/main/java/io/sentry/samples/jul/Main.java:27]: This sample has to be started with `../../gradlew run` as there must be a system property `Djava.util.logging.config.file` pointing to `logging.properties` file set.

File: sentry-samples/sentry-samples-log4j2/src/main/resources/sentry.properties (hunk 2 above)
Comment (maciejwalkowiak) [reply sentry-samples/sentry-samples-log4j2/src/main/resources/sentry.properties:2]: There's an incorrect DSN in log4j2.xml. Change it to `https://502f25099c204a2fbf4cb16edc5975d1@o447951.ingest.sentry.io/5428563` and events show up.

File: sentry-samples/sentry-samples-spring/src/main/resources/sentry.properties (hunk 3 above)
Comment (maciejwalkowiak) [reply sentry-samples/sentry-samples-spring/src/main/resources/sentry.properties:3]: It does but there is a bug with merging options, I'll fix it in a minute.

File: sentry-samples/sentry-samples-jul/src/main/java/io/sentry/samples/jul/Main.java (hunk 4 above)
Comment (marandaneto) [reply sentry-samples/sentry-samples-jul/src/main/java/io/sentry/samples/jul/Main.java:27]: @maciejwalkowiak yep, its on the readme, I've used this command

File: sentry-samples/sentry-samples-jul/src/main/resources/sentry.properties (hunk 1 above)
Comment (marandaneto) [reply sentry-samples/sentry-samples-jul/src/main/resources/sentry.properties:6]: @maciejwalkowiak is it related to the DSN as well?

Comment (marandaneto) [reply sentry-samples/sentry-samples-jul/src/main/resources/sentry.properties:6]: looks like same as https://github.com/getsentry/sentry-java/pull/1282/files#r582653481

**Review by maciejwalkowiak — APPROVED** [review]

**Pull request merged**